rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -750,6 +750,22 @@ error_code sceNpMatching2GetRoomSlotInfoLocal(SceNpMatching2ContextId ctxId, con
 		return SCE_NP_MATCHING2_ERROR_NOT_INITIALIZED;
 	}
 
+	roomSlotInfo->roomId        = roomId;
+	roomSlotInfo->publicSlotNum    = 2; // TODO
+	roomSlotInfo->passwordSlotMask   = 0; // TODO
+	roomSlotInfo->privateSlotNum     = 0; // TODO
+	roomSlotInfo->openPrivateSlotNum = 0; // TODO
+
+	const std::set<SceNpMatching2RoomMemberId> member_list = nph.room_members[roomId];
+	const u32 num_members = static_cast<u32>(member_list.size());
+
+	if (num_members == 0)
+		roomSlotInfo->openPublicSlotNum = 0;
+	else
+		roomSlotInfo->openPublicSlotNum = (1 << num_members) - 1; // Set bitsmask bit 1 for all members
+
+	roomSlotInfo->openPublicSlotNum = roomSlotInfo->publicSlotNum - num_members;
```

Comment:
looking at this upper part, it makes no sense at all, since openPublicSlotNum will be assigned a new value afterwards anyway

Reply:
copy pasta error, should be `joinedSlotMask` thx